findbugs.properties

```diff
@@ -30,7 +30,12 @@ defaults.mavenArtifactId = sonar-findbugs-plugin
 4.2.6.changelogUrl = https://github.com/spotbugs/sonar-findbugs/releases/tag/4.2.6
 4.2.6.downloadUrl = https://repo.maven.apache.org/maven2/com/github/spotbugs/sonar-findbugs-plugin/4.2.6/sonar-findbugs-plugin-4.2.6.jar
 4.2.7.description = Use SpotBugs 4.8.3, sb-contrib 7.6.4, and findsecbugs 1.12.0
-4.2.7.sqVersions = [9.9,LATEST]
+4.2.7.sqVersions = [9.9,10.4]
```

Comment:
Yes, this range now needs to end at  `10.4.1` or `10.4.*`

Reply:
Thanks Ann, I've made the change